client/blocks/reader-post-card/conversation-post.jsx

```diff
@@ -3,27 +3,49 @@
  */
 import React from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { InView } from 'react-intersection-observer';
 
 /**
  * Internal Dependencies
  */
 import ConversationPostList from 'blocks/conversations/list';
 import CompactPostCard from 'blocks/reader-post-card/compact';
+import { setViewFeedPost, unsetViewFeedPost } from 'state/reader/viewing/actions';
+import { getSite } from 'state/sites/selectors';
 
 class ConversationPost extends React.Component {
 	static propTypes = {
 		post: PropTypes.object.isRequired,
 		commentIds: PropTypes.array.isRequired,
 	};
 
+	toggleItemView = inView => {
+		const { post } = this.props;
```

Comment:
```suggestion
		const { post, setViewFeedPost, unsetViewFeedPost } = this.props;
```

Reply:
I don't think that is possible, it will conflict with the import.